Source: Upstream git commit
Upstream: yes, https://github.com/ldo/dvdauthor/commit/c82aaa4eb1a1c36bf7e2b7ae3c9140d0bf8000b5
Reason: Fix dvdauthor build against libpng-1.5

From c82aaa4eb1a1c36bf7e2b7ae3c9140d0bf8000b5 Mon Sep 17 00:00:00 2001
From: Lawrence D'Oliveiro <ldo@geek-central.gen.nz>
Date: Wed, 26 Jan 2011 00:48:37 +0000
Subject: [PATCH] fix to build against png-1.5 (thanks to Thomas Klausner)

---
 src/spuunmux.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/spuunmux.c b/src/spuunmux.c
index 5e565d1..9ad2371 100644
--- a/src/spuunmux.c
+++ b/src/spuunmux.c
@@ -39,6 +39,7 @@
 #include <netinet/in.h>
 
 #include <png.h>
+#include <zlib.h>
 
 #include "rgb.h"
 #include "common.h"
@@ -610,7 +611,7 @@ static int write_png
         png_destroy_write_struct(&png_ptr, (png_infopp)NULL);
         return -1;
       } /*if*/
-    if (setjmp(png_ptr->jmpbuf))
+    if (setjmp(png_jmpbuf(png_ptr)))
       {
         png_destroy_write_struct(&png_ptr, &info_ptr);
         fclose(fp);
-- 
1.7.10.3

